src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -1414,7 +1414,6 @@ pub struct CreateContinualTaskStatement<T: AstInfo> {
 pub enum ContinualTaskStmt<T: AstInfo> {
     Delete(DeleteStatement<T>),
     Insert(InsertStatement<T>),
-    // TODO(ct): Update/upsert?
```

Comment:
Doesn't feel actionable IMO. I mostly left this in here in first place without thinking because when I type an enum with Delete and Insert variants, I naturally consider Update/Upsert. Notably, none of the initial use cases in my doc seemed to need them